pkg/controller/plan/kubevirt.go

```diff
@@ -834,6 +834,56 @@ func (r *KubeVirt) EnsureGuestConversionPod(vm *plan.VMStatus, vmCr *VirtualMach
 	return
 }
 
+func (r *KubeVirt) EnsureVirtV2VPVCStatus() (ready bool, err error) {
```

Comment:
the name of this function is not specific to OVA but inside we set `"ova": "nfs-pvc"`  so the name should change to reflect that